### DISCUSSION TITLE
GB/T 7714—2015（著者-出版年，双语，无 URL、DOI）可否将文内注显示2个作者（3个及以上替换为等），参考文献展示全部作者

### DISCUSSION THREAD

**Discussion by gusang123** [zotero-chinese/wiki#290] [常规 / General]
一个小小的建议，也是希望能求助一下大家，就是对于上述版式，可否略做调整：将文内注中作者在三个以下的全部展示（A、B，2005），三个及以上只展示一个＋等（A 等，2005）；参考文献、展示全部作者，而非三个以上以等替代。

**Comment by zepinglee** [comment]
具体哪个期刊/学校的格式？

国标的描述不够清晰，也没有示例，但是推测其大意为引注中两个或以上作者显示为”张三 等“或”Doe et al.“。

<img width="798" alt="Screenshot 2024-10-05 at 10 10 10" src="https://github.com/user-attachments/assets/84b23c11-36a3-4743-b9b2-8185550d6ba7">

**Reply by gusang123** [reply]
非常感谢您的回复~确实关于国标没有明确的要求。我是在参考《社会学研究》和《社会》两本杂志基础上提出的一些想法。其中《社会学研究》已经有版式，但其版式对页码缺少规范，所以结合了《社会》关于文中注和参考文献的注法。感觉结合起来，对社会学领域会更通用一些~
﹣《社会》参考文献要求：
（1）中文文献与外文文献分别排列，中文在前、外文在后，以作者姓氏英文字母A - Z为序排列，中文译名以译名的首个字母为准。
（2）外文作者（无论原名或译名）均须写出完整的姓和名，中间名可缩略表示；多位作者的，第一作者按照“姓，名”格式，其余作者按照外文姓名的正常顺序，即“名 姓”，不同作者姓名之间用“，”隔开，最后两位姓名之间用“and”连接。
（３）**若一篇文献只有两位作者，两位作者的姓名在文中夹注中均须写明**；若有两位以上者，则文中夹注可只写明第一作者姓名，后加“等”或“et al.”缩略表示，但**须在相应参考文中写明所有作者的姓名**。
（４）中文文献应标明文献类型（著作为［M］，期刊为［J］，汇编为［G］，学位论文为［D］）；外文文献中的论文用引号标明，著作（包括文集）及期刊用斜体标明；参考文献的格式为：“作者名．出版年份．文献名称［文献类型］．文献出处（卷期次）．出版地：出版者．”；中外文论文应在期数后标明起止页码。具体如下：
李培林、田丰．2012．中国农民工社会融入的代际比较［J］．社会 32（5）：1-24．
徐勇、吴毅．2001．乡土中国的民主选举［M］．武汉：华中师范大学出版社．
Kim，Illsoo．1981．New Urban Immigrants：The Korean Community in New York.Prinveton．N.J.：Princton University Press．
Gallo，William T．，Jennie E．Brand，Hsun-Mei Teng，Linda Leo-Summers，and Amy L．Byers．2009．“Diffenential Impact of Involuntary Job Loss on Physical Disability amang Older Workers：Does Predisposition Matter？”Research on Aging 31（3）：345－360．

**Reply by zepinglee** [reply]
按照“样式申请”的模板在 https://github.com/zotero-chinese/styles/issues 发 issue，填写样式信息。

**Reply by zepinglee** [reply]
> 其中《社会学研究》已经有版式，但其版式对页码缺少规范

具体是怎么“对页码缺少规范”？建议给出具体文献的示例。

> 感觉结合起来，对社会学领域会更通用一些

仓库 <https://github.com/citation-style-language/styles> 的原则是，对国标的实现尽可能忠于原文。像「文内注显示2个作者」和「参考文献展示全部作者」这些都很容易，所以没有必要额外放一个版本。

至于社会学领域，我目前实现了《综合性期刊文献引证技术规范》的「[注释](https://zotero-chinese.com/styles/citation-specifications-for-comprehensive-journals-note/)」和「[著者—出版年](https://zotero-chinese.com/styles/citation-specifications-for-comprehensive-journals-author-date/)」的 CSL 样式，以及[《中国社会科学》样式](https://zotero-chinese.com/styles/social-sciences-in-china/)，应该足够用了。
